src/Compilers/CSharp/Portable/Binder/Binder_Statements.cs

```diff
@@ -1542,6 +1542,12 @@ private BoundExpression InferTypeForDiscardAssignment(BoundDiscardExpression op1
                 {
                     if (isRef)
                     {
+                        // https://github.com/dotnet/csharplang/blob/main/proposals/low-level-struct-improvements.md#rules-ref-reassignment
```

Comment:
nit: Link can now be updated to csharp-11.0 sub-folder

Reply:
Thanks. It looks like there are several references to fix up. I'll address these in a separate PR.